rhobs/alerting/data_plane/prometheus.gitops_deployment_alerts.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-gitops-deployment-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: gitops_deployment_alerts
+    interval: 1m
+    rules:
+    - alert: GitOpsDeploymentHealthErrors
+      expr: |
+        (
+          sum(kube_deployment_status_replicas_ready{namespace="gitops-service-argocd"})
```

Comment:
Do we need to `sum` here?

Reply:
Here we get series for multiple deployments, to handle that I used `SUM()`.

Reply:
It is supposed to show 4 rows.

![image](https://github.com/redhat-appstudio/o11y/assets/63054462/87351e6b-c508-4b9a-b046-0719c0ac625c)

Reply:
What cluster is this taken from?

Reply:
It is taken from RHTAP Multi Tenant cluster.